packages/discord.js/src/structures/ReactionCollector.js

```diff
@@ -115,6 +114,7 @@ class ReactionCollector extends Collector {
    * @returns {?(Snowflake|string)}
    * @private
    */
+  // eslint-disable-next-line no-unused-vars
   collect(reaction, user) {
```

Comment:
With the old docs I believe having this comment here would disconnect the JSDoc from the method
Is this no longer an issue with the new docs?

Reply:
just tested, documentation shows up fine